app/main/views/drafts.py

```diff
@@ -36,16 +39,19 @@ def create_draft_service(service_id):
         abort(400, "Draft already exists for service {}".format(service_id))
 
     draft = DraftService.from_service(service)
+
+    db.session.add(draft)
+    db.session.flush()
+
     audit = AuditEvent(
         audit_type=AuditTypes.create_draft_service,
         user=updater_json['updated_by'],
         data={
+            "draft_id": draft.id,
```

Comment:
Audit events in `service_utils` were changed to use the `serviceId` as key, we should probably change this one to match the new key convention